### DISCUSSION TITLE
No action for creating C/CPP programs?

### DISCUSSION THREAD

**Discussion by NorskieRider** [codefori/vscode-ibmi#1100] [IBM i] [answered]
I noticed that there wasn't an 'action' for compiling c/cpp programs (crtbndc, crtcmod, crtcppmod, etc). 

Did I miss something during the installation or are these missing on purpose?

**Comment by worksofliam** [comment, marked as answer]
Hey. There aren't any defaults for those most likely. You can of course create your own, and we're open to PRs incase you'd like to add the defaults!
